### PR TITLE
feat: allow db cert specification

### PR DESCRIPTION
**Pull request by lauhon**
Hey @brunojppb,

thanks for putting this project togehter - super cool!

Digital Ocean Databases for example are per default exposed with self signed certificates. It would be really handy to allow those to be configured.

I have pretty much no Idea what Im doing, but got some inspiration [here](https://github.com/elixir-ecto/postgrex/issues/463#issuecomment-647454607)

### CODE REVIEW

**Comment by lauhon** [comment]
Hey @brunojppb, any chance you could give your opinion on this?

File: config/runtime.exs [hunk 1]
```diff
@@ -53,6 +53,12 @@ if config_env() == :prod do
 
   config :turbo, Turbo.Repo,
     ssl: use_db_ssl,
+    ssl_opts: [
+      verify: :verify_peer,
+      cacertfile: System.get_env("DATABASE_CA_CERT"),
+      verify_fun: &:ssl_verify_hostname.verify_fun/3,
+      server_name_indication: String.to_charlist(System.get_env("DATABASE_HOST", "")),
+    ],
```
Comment (brunojppb) [inline config/runtime.exs:61]: Let's make this a configuration based on the DB cert env variable presence:

```suggestion
  db_ssl_opts = if System.get_env("DATABASE_CA_CERT") do
   [
      verify: :verify_peer,
      cacertfile: System.get_env("DATABASE_CA_CERT"),
      verify_fun: &:ssl_verify_hostname.verify_fun/3,
      server_name_indication: String.to_charlist(System.get_env("DATABASE_HOST", "")),
    ]
  else
   []
  end


  config :turbo, Turbo.Repo,
    ssl: use_db_ssl,
    ssl_opts: [],
```

**Review by brunojppb — COMMENTED** [review]
Hey @lauhon,
thanks for contributing. For some reason, I missed the notification when you raised this PR.

Hey let's try this out.
Let's make this a configuration instead so it can be decided at runtime.

**Review by brunojppb — APPROVED** [review]

**Comment by brunojppb** [comment]
@lauhon new Docker image available here: 
https://github.com/brunojppb/turbo-racer/pkgs/container/turbo-racer/316018867?tag=1.6.0

**Pull request merged**